Modules/posixmodule.c

```diff
@@ -5467,6 +5467,41 @@ os__path_islink_impl(PyObject *module, PyObject *path)
 #endif /* MS_WINDOWS */
 
 
+/*[clinic input]
+os._path_splitroot_ex
+
+    p: path_t
+
+[clinic start generated code]*/
```

Comment:
What do we do with the docstring here? There's no longer a Python wrapper around this function.

Reply:
Worth adding a one liner for what it does, or at least the return value. It'll still be accessible from Python, and a short docstring is not the same as documented.